### DISCUSSION TITLE
[General]GLPI-Agent release changelog

### DISCUSSION THREAD

**Discussion by sebma** [glpi-project/glpi-agent#431] [General]
### Your question

Hi, what are the changes of the new GLPI-Agent release ?

**Comment by g-bougard** [comment]
Hi @sebma 

most important ones are described in the announcement: https://glpi-project.org/glpi-agent-1-5/

The detailed changes are always listed in the Changes file:
https://github.com/glpi-project/glpi-agent/blob/1.5/Changes

**Comment by sebma** [comment]
@g-bougard Just what I needed, thanks 👍 .
